content/team/mattR.md

```diff
@@ -1,6 +1,6 @@
 ---
 name : Matthew Robinson
-title : Site Reliability Architect
+title : Sr. Site Reliability Engineer
```

Comment:
Perhaps this update is not needed and you can just remove it, because unfortunately, Matt is leaving Storj tomorrow.